synthtool/gcp/templates/node_library/README.md

```diff
@@ -0,0 +1,117 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
```

Comment:
this is almost a direct port from:

https://github.com/GoogleCloudPlatform/nodejs-repo-tools/blob/master/templates/lib_readme.tpl